x-pack/plugins/siem/cypress/integration/cases.spec.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { case1 } from '../objects/case';
+
+import {
+  ALL_CASES_CLOSE_ACTION,
+  ALL_CASES_CLOSED_CASES_COUNT,
+  ALL_CASES_CLOSED_CASES_STATS,
+  ALL_CASES_COMMENTS_COUNT,
+  ALL_CASES_DELETE_ACTION,
+  ALL_CASES_NAME,
+  ALL_CASES_OPEN_CASES_COUNT,
+  ALL_CASES_OPEN_CASES_STATS,
+  ALL_CASES_OPENED_ON,
+  ALL_CASES_PAGE_TITLE,
+  ALL_CASES_REPORTER,
+  ALL_CASES_REPORTERS_COUNT,
+  ALL_CASES_SERVICE_NOW_INCIDENT,
+  ALL_CASES_TAGS,
+  ALL_CASES_TAGS_COUNT,
+} from '../screens/all_cases';
+import {
+  ACTION,
+  CASE_DETAILS_DESCRIPTION,
+  CASE_DETAILS_PAGE_TITLE,
+  CASE_DETAILS_PUSH_AS_SERVICE_NOW_BTN,
+  CASE_DETAILS_STATUS,
+  CASE_DETAILS_TAGS,
+  CASE_DETAILS_TIMELINE_MARKDOWN,
+  CASE_DETAILS_USER_ACTION,
+  CASE_DETAILS_USERNAMES,
+  PARTICIPANTS,
+  REPORTER,
+  USER,
+} from '../screens/case_details';
+import { TIMELINE_DESCRIPTION, TIMELINE_QUERY, TIMELINE_TITLE } from '../screens/timeline';
+
+import { goToCaseDetails, goToCreateNewCase } from '../tasks/all_cases';
+import { openCaseTimeline } from '../tasks/case_details';
+import { backToCases, createNewCase } from '../tasks/create_new_case';
+import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
+import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
+
+import { CASES } from '../urls/navigation';
+
+describe('Cases', () => {
+  before(() => {
+    esArchiverLoad('timeline');
```

Comment:
I'm not sure this is working, the timeline is not showing up in the search
<img width="1400" alt="Screen Shot 2020-04-20 at 12 17 49 PM" src="https://user-images.githubusercontent.com/6935300/79785354-f7de4880-8300-11ea-890d-277584983ac0.png">

Reply:
ok we're not mocking this timeline data at all

Reply:
see my comments in `x-pack/plugins/siem/cypress/tasks/create_new_case.ts` for more on this

Reply:
The Cypress tests are written now thinking that are going to be executed in CI mode or in Jenkins what means that the data is populated using es_archiver utility. 

For this test the timeline used in the test is populated using es_archiver.